.gitlab/test_nixlbench.sh

```
@@ -76,23 +66,53 @@ else
+if $HAS_GPU ; then
+    for op_type in READ WRITE; do
+        for initiator in $seg_types; do
+            for target in $seg_types; do
+                run_nixlbench_two_workers_asio --backend UCCL --op_type $op_type --initiator_seg_type $initiator --target_seg_type $target --check_consistency
+            done
+        done
+    done
+fi
+
+run_nixlbench_two_workers_etcd() {
+    args="$@"
```

Shellcheck (0.11.0)

[warning] Assigning an array to a string! Assign as array, or use * instead of @ to concatenate.

(SC2124)